test/Table_Tests/src/Common_Table_Operations/Join/Union_Spec.enso

```diff
@@ -25,408 +25,420 @@ type Data
         Data.Value (create_connection_fn Nothing)
 
     teardown self = self.connection.close
-
+call_static_union tables =
+    Table.from_union tables ...
+call_member_union tables =
+    first = tables.first
+    rest = tables.drop 1
+    first.union rest ...
```

Comment:
Since it's a not-often-used trick, we may want to leave a note in the comment that `...` is used here to allow passing of additional optional arguments through.

Reply:
Added a comment to document this a bit